### PR TITLE
Updating demos doc page

### PR DESCRIPTION
**Pull request by bmmpxf**

### CODE REVIEW

**Comment by jodygarnett** [comment]
The doc/en/user/source/configuration/demos/img/demos_reprojectionconsole.png looks squashed / wrong aspect ratio.

File: doc/en/user/source/configuration/demos/index.rst [hunk 1]
```diff
@@ -60,11 +76,62 @@ The :guilabel:`Code` column refers to the unique integer identifier defined by t
...
+
+
+.. _demos_wpsrequestbuilder:
+
+WPS Request Builder
+-------------------
+
+GeoServer with the WPS extension installed includes a request builder for generating and executing WPS processes. Since WPS requests can be cumbersome to author, this tool can make working with WPS much easier.
+
+Read more about the :ref:`wps_request_builder`.
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/demos/index.rst]: So this is over in the WPS docs?

File: doc/en/user/source/configuration/demos/index.rst [hunk 2]
```diff
@@ -60,11 +76,62 @@ The :guilabel:`Code` column refers to the unique integer identifier defined by t
...
+
+You can select the version of WCS you wish to work with, as well as any valid layer published in GeoServer.
+
+.. figure:: img/demos_wcsrequestbuilder.png
+
+   WCS Request Builder
+
+Once the parameters are set, you can select :guilabel:`GetCoverage` to execute the WCS request, or :guilabel:`Generate GetCoverage XML` to view/save the actual request XML.
+
+Read more about :ref:`wcs`.
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/demos/index.rst]: Can we add a page for the WCS Request builder and link to that?

Comment (jodygarnett) [inline doc/en/user/source/configuration/demos/index.rst]: okay, merging then.

**Comment by bmmpxf** [comment]
Thanks @jodygarnett . Should I backport this anywhere?

**Pull request merged**